src/google/protobuf/unittest.proto

```diff
@@ -76,6 +76,13 @@ message TestAllTypes {
     NEG = -1;  // Intentionally negative.
   }
 
+  enum NestedEnumAllowingAlias {
```

Comment:
I try not to modify the core protos since the c++ tests drive them, looks like you can use https://github.com/google/protobuf/blob/master/src/google/protobuf/unittest.proto#L518-L526 instead.